src/Notification.js

```diff
@@ -134,6 +137,7 @@ class Notification extends Component {
           icon={icon}
           vibrate={vibrate}
           onClose={() => this.setState({ isOpen: false }, this.closeNotification)}
+          {...this.state.passProps}
```

Comment:
Hey! Sorry about the late response. Happy for this feature to go in with a couple of few changes.
1. First can we rename `passProps` to `additionalProps`, I think that language is clearer about what they are.
2. Secondly can you not spread them and instead pass them like `additionalProps={additionalProps}`, I'd like to keep the top level props scope clean from arbitrary props.
3. Can you add this new prop to the README and document it.